lib/hmmlearn/hmm.py

```diff
@@ -359,8 +359,7 @@ class MultinomialHMM(BaseHMM):
     MultinomialHMM(algorithm='viterbi',...
     """
 
-    # TODO: accept the prior on emissionprob_ for consistency.
-    def __init__(self, n_components=1,
+    def __init__(self, n_components=1, emissionprob_prior=1.0,
```

Comment:
This means that anyone who previously passed more than one positional argument will now accidentally set emissionprob_prior instead of startprob_prior.  While such a use of the API seems quite unlikely, you can avoid anyone silently getting wrong results by making all parameters except for n_components keyword-only (with a `*`) -- now at least anyone with the old usage will have an exception and will know they need to update their code.

Also, for consistency with GaussianHMM, emissionprob_prior should come after startprob_prior and transmat_prior.

Other than that, this looks good to me.

Reply:
Good point, I did think about that but I didn't have the solution, thanks for that!